config/routes.rb

```diff
@@ -1,9 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-  # resources :trivia, only: :index
+  
+  match '*unmatched', to: 'application#route_not_found', via: :all 
```

Comment:
I'm not 100% sure what this is doing, let's discuss before I merge

Reply:
One note: I did move this line to the end of the file as we would want to to execute only after all other possible routes are evaluated against the request.